lib/exla/tensor.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Exla.Tensor do
+  alias __MODULE__, as: Tensor
+  alias Exla.Client
+  alias Exla.Shape
+  # TODO: Different namespace?
```

Comment:
Do you mean if the struct fields should be in another namespace? No, I think this is the correct place. :)

Reply:
I meant the Tensor module as a whole. Just because everything else wraps something specific in XLA...more of something to think about later on separating XLA specifics from the higher level stuff

Reply:
Ah, sure!